src/CMakeLists.txt

```diff
@@ -320,7 +320,7 @@ if(BUILD_DAEMON)
   )
   list(APPEND installable_targets bitcoind)
 endif()
-if(WITH_MULTIPROCESS)
+if(WITH_MULTIPROCESS AND BUILD_DAEMON)
```

Comment:
Would it be possible to use `bitcoin_daemon_status` here somehow, instead of repeating the expression ? (just a small maintainability thing to keep the reporting and usage consistent if it changes in the future)

Reply:
It turns out yes, as long as you define `bitcoin_daemon_status` before `add_subdirectory(src)`. Pushing that.

Reply:
TBH, I'd refrain from making `bitcoin_daemon_status` a "global" variable.

Reply:
In that case I'm inclined to not use the variable at all.

Reply:
Oh but `message()` can't handle that. Alright, let's go back to the previous version.

Reply:
> In that case I'm inclined to not use the variable at all.

Ok, yes, that would have been my other proposal. But a variable seems to be needed then, let's leave it as-is.